src/libraries/System.Private.CoreLib/src/System/Threading/Tasks/Task.cs

```diff
@@ -6828,7 +6828,7 @@ public enum TaskContinuationOptions
 
     // Special internal struct that we use to signify that we are not interested in
     // a Task<VoidTaskResult>'s result.
-    internal struct VoidTaskResult { }
+    internal readonly struct VoidTaskResult { }
```

Comment:
We have a few of these "empty structs" throughout the BCL that are basically just markers for the type system.

Is it something we may want to centralize or expose more generally to help with trimming and to avoid the need for extra metadata?

Reply:
If we have multiple in corelib, seems fine to collapse them down.  We could have a separate conversion about exposing them publicly, or better yet, allowing System.Void to be used :)